Open WorldCat.js

```diff
@@ -341,7 +340,17 @@ function fetchIDs(isbns, ids, callback) {
 				} else {
 					Z.debug("Could not extract OCLC ID for ISBN " + isbn);
 				}
-			} else {
+			}
+			//but sometimes we have single items
+			else if (getFirstContextObj(doc)) {
```

Comment:
The `getFirstContextObj` is AFAIK just checking for the presence of COinS data, but this is also available in search pages, e.g. http://www.worldcat.org/search?qt=worldcat_org_all&q=mannheim